### ISSUE TITLE
crouton crashes whenever i open any app inside crouton

### ISSUE THREAD

**Issue by drilltimepoppa** [dnschneid/crouton#4679]
```
Please paste the output of the following command here: sudo edit-chroot -all
```

#### Please describe your issue:
whenever i open chrome or anything in crouton with any release, it crashes to the dev mode screen where it says "os verification is off"

#### If known, describe the steps to reproduce the issue:
i think this is only me with this issue lmao

**Comment by DennisLfromGA** [comment]
We'll need for you to show us the output of `sudo edit-chroot -all` as requested before can help troubleshoot you issue.

-**DennisLfromGA**

**Comment by drilltimepoppa** [comment]
> We'll need for you to show us the output of `sudo edit-chroot -all` as requested before we can help troubleshoot you issue.
> 
> -**DennisLfromGA**

chronos@localhost / $ sudo edit-chroot -al
name: xenial
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/xenial...
crouton: version 1-20220111101058~master:a9eafaee
release: xenial
architecture: amd64
xmethod: xorg
targets: xfce
host: version 14324.72.0 (Official Build) stable-channel grunt 
kernel: Linux localhost 5.10.77-12631-gd9f58bbd77d0 #1 SMP PREEMPT Wed Jan 12 19:44:11 PST 2022 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/xenial...

**Comment by DennisLfromGA** [comment]
@drilltimepoppa,

Thanx for the info.  

I only see one release, 'xenial', one xmethod, 'xorg', and one desktop target, 'xfce'.
I would try adding xmethod 'xiwi' and/or another desktop target and/or another release and see if you get any further.

I use 'xenial' with no issues on my device, here are the details:
```
----------------------------------------------------------------------
chroot name: xenial
crouton: version 1-20210907212903~master:9b281df5
release: xenial
architecture: amd64
xmethod: xiwi
targets: xiwi,xorg,cli-extra,keyboard,touch,gnome-desktop,lxde-desktop,unity-desktop,xfce-desktop,chrome-beta
host: version 14388.27.0 (Official Build) beta-channel eve 
kernel: Linux localhost 5.4.163-17375-g176255512998 #1 SMP PREEMPT Mon Jan 10 22:16:53 PST 2022 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
----------------------------------------------------------------------
```
I also use bionic & focal with similar methods & targets with no issue.

*Hope this helps,*
-**DennisLfromGA**

**Comment by magixuser** [comment]
is it possible to install impish on Samsung XE303C12?

**Comment by DennisLfromGA** [comment]
@magixuser,

The 'impish' distro on a Samsung XE303C12 has absolutely *nothing* to do with:  
- #4679

Please do your own searches and/or post a new issue if you can't find an answer elsewhere.

-**DennisLfromGA**

**Comment by magixuser** [comment]
> 

pls help me with #4681

**Comment by drilltimepoppa** [comment]
> @drilltimepoppa,
> 
> Thanx for the info.
> 
> I only see one release, 'xenial', one xmethod, 'xorg', and one desktop target, 'xfce'. I would try adding xmethod 'xiwi' and/or another desktop target and/or another release and see if you get any further.
> 
> I use 'xenial' with no issues on my device, here are the details:
> 
> ```
> ----------------------------------------------------------------------
> chroot name: xenial
> crouton: version 1-20210907212903~master:9b281df5
> release: xenial
> architecture: amd64
> xmethod: xiwi
> targets: xiwi,xorg,cli-extra,keyboard,touch,gnome-desktop,lxde-desktop,unity-desktop,xfce-desktop,chrome-beta
> host: version 14388.27.0 (Official Build) beta-channel eve 
> kernel: Linux localhost 5.4.163-17375-g176255512998 #1 SMP PREEMPT Mon Jan 10 22:16:53 PST 2022 x86_64 x86_64 x86_64 GNU/Linux
> freon: yes
> ----------------------------------------------------------------------
> ```
> 
> I also use bionic & focal with similar methods & targets with no issue.
> 
> _Hope this helps,_ -**DennisLfromGA**

hey so i added the "xiwi" xmethod with the crouton intergration extension and im getting the same thing. im going to try a diffrent desktop

**Comment by Vaggysag** [comment]
Same issue, any luck?
